services/web-app/components/footer/footer.js

```diff
@@ -4,71 +4,104 @@
-            <a className={styles.link}>{'Terms of use'}</a>
-          </Link>
-          <Link href="https://www.google.com">
-            <a className={styles.link}>{'Ibm.com'}</a>
-          </Link>
-        </Column>
-        <Column className={clsx(styles.column, styles.columnBorderTop)} sm={4} md={2} lg={4}>
-          <Link href="https://www.google.com">
-            <a className={styles.link}>{'Medium'}</a>
-          </Link>
-          <Link href="https://www.google.com">
-            <a className={styles.link}>{'Twitter'}</a>
-          </Link>
-        </Column>
-        <Column
-          className={clsx(styles.column, styles.columnBorderTop)}
-          sm={4}
-          md={3}
-          lg={4}
-          xlg={3}
-        >
-          <p className={styles.text}>
-            {'Have questions? '}
-            <Link href="https://www.google.com">
-              <a className={clsx(styles.link, styles.linkUnderline)}>{'Email'}</a>
-            </Link>
-            {' us for site feedback or open an issue in '}
-            <Link href="https://www.google.com">
-              <a className={clsx(styles.link, styles.linkUnderline)}>{'Github.'}</a>
-            </Link>
-          </p>
-        </Column>
-      </Grid>
-      <Grid>
-        <Column sm={4} md={6} lg={6} xlg={6}>
-          <IbmCloud className={styles.icon} size={64} />
-        </Column>
-      </Grid>
-    </div>
+    <Theme theme="g100" className={styles.container}>
+      <footer>
+        <Grid className={styles.footer}>
+          <Column sm={4} md={2} lg={{ span: 2, start: 5 }} className={styles.column}>
+            <ul>
+              {colOne.map((item, i) => (
+                <li key={item}>
+                  <Link href={item.link} key={i}>
+                    <a className={clsx(styles.text, styles.link, styles.listLink)}>{item.text}</a>
+                  </Link>
+                </li>
+              ))}
+            </ul>
+          </Column>
+          <Column sm={4} md={2} lg={4} className={clsx(styles.column)}>
+            <ul>
+              {colTwo.map((item, i) => (
+                <li key={item}>
+                  <Link href={item.link} key={i}>
+                    <a className={clsx(styles.text, styles.link, styles.listLink)}>{item.text}</a>
+                  </Link>
+                </li>
+              ))}
+            </ul>
+          </Column>
+          <Column sm={4} md={3} lg={4} className={styles.column}>
+            <p className={styles.text}>
+              Have questions?{' '}
+              <Link href="https://github.com/carbon-design-system/carbon-platform">
+                <a className={clsx(styles.text, styles.link)}>Email</a>
+              </Link>{' '}
+              us for site feedback or open an issue in{' '}
+              <Link href="https://github.com/carbon-design-system/carbon-platform">
+                <a className={clsx(styles.text, styles.link)}>Github.</a>
+              </Link>
+            </p>
```

Comment:
Do we want to add the copyright here?

`const currentYear = new Date().getFullYear();`

`<p>Copyright © {currentYear} IBM</p>`

Reply:
Yep! I spoke with @jeanservaas and this is what will be the content: 

Last updated March 2022 (current month & year)
©2022 IBM Design Program Office